cmd/crc/cmd/constants.go

```diff
@@ -1,7 +1,8 @@
 package cmd
 
 const (
-	commandName      = "crc"
-	descriptionShort = "Local OpenShift 4.x cluster"
-	descriptionLong  = "Deploy local instance of an OpenShift 4.x cluster for development purpose"
+	commandName                  = "crc"
+	descriptionShort             = "Local OpenShift 4.x cluster"
+	descriptionLong              = "Deploy local instance of an OpenShift 4.x cluster for development purpose"
+	machineNotExistsErrorMessage = "Machine \"crc\" does not exist. Use \"crc start\" to add a new one."
```

Comment:
"nonExistingMachineErrorMessage", or "missingMachineErrorMessage" is better I think. But if the machine existence check is moved to a helper, we don't have to define a constant for the error message.